Content.Shared/PowerCell/SharedPowerCellSystem.cs

```diff
@@ -58,7 +58,10 @@ private void OnCellRemoved(EntityUid uid, PowerCellSlotComponent component, EntR
 
     private void OnCellSlotInit(EntityUid uid, PowerCellSlotComponent component, ComponentInit args)
     {
-        _itemSlotsSystem.AddItemSlot(uid, CellSlotContainer, component.CellSlot);
+        if (_itemSlotsSystem.TryGetSlotById(uid, component.CellSlotId, out var itemSlot))
+        {
+            component.CellSlot = itemSlot;
+        }
 
         if (string.IsNullOrWhiteSpace(component.CellSlot.Name) &&
             !string.IsNullOrWhiteSpace(component.SlotName))
```

Comment:
The `component.SlotName` should just be removed in favor of using the slots' name field. I'm not even sure why this was here in the first place.